src/Symfony/Component/ErrorHandler/DebugClassLoader.php

```diff
@@ -127,7 +127,7 @@ public function __construct(callable $classLoader)
         $this->patchTypes += [
             'force' => null,
             'php' => \PHP_MAJOR_VERSION.'.'.\PHP_MINOR_VERSION,
-            'deprecations' => false,
+            'deprecations' => \PHP_VERSION_ID >= 70300,
```

Comment:
Why 7.3? That feels a bit arbitrary and I'd like to understand it. 🙂

Reply:
My reasoning is that this might trigger deprecations that some libs might not be prepared to deal with. I think this should reduce what these older libs would mostly dismiss as noise otherwise.

Reply:
But 7.3 didn't change anything relevant on the type system, did it? 🤔 Contra-variant return types were introduced with 7.4 and unions/mixed with 8.0.

Reply:
I'm fine bumping to 7.4 if that's what you mean :)

Reply:
bumped